phylogenetic/defaults/clades.tsv

```diff
@@ -5,6 +5,9 @@ outgroup	nuc	179226	T
 clade I	nuc	87560	T
 clade I	nuc	136015	A
 
+clade Ib	nuc	6014	G
```

Comment:
Note that this will now label clade-Ib on our canonical all-mpox build. I didn't include it in the Nextclade dataset. (Interestingly that uses different clade-defining mutations, but that's tangential to my work here.)